src/server/job_recov_db.c

```diff
@@ -265,7 +238,7 @@ db_to_svr_job(job *pjob,  pbs_db_job_info_t *dbjob)
  *		Load data from server resv object to a database resv object
  *
  * @see
- * 		resv_save_db
```

Comment:
i would actually call out that we are saving to db, so i would not change the name to resv_save

Reply:
I just renamed it to give the same look like resv_alloc() and resv_free(), I will revert it back to _db